appveyor.yml

```diff
@@ -35,7 +35,7 @@ build_script:
     msbuild %libusb_2012% /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll"
 
     msvc/appveyor.bat
-
+    
```

Comment:
Why do you add spaces here?

I guess your changes in appveyor.yml are not needed and made by mistake.